samples/ios/app/Cartfile

```diff
@@ -1,2 +1,2 @@
 github "httpswift/swifter" ~> 1.4.7
-github "mozilla/glean" "master"
+github "travis79/glean" "ios-debug-tools"
```

Comment:
This is just for testing and needs to be reverted before merging.

Reply:
Hmpf, I really dislike that it has to build against master.

Reply:
I can pin it to a version when I revert this...

Reply:
Nah, I mean more the fact that we can't just point to "this directory two layers above".
With this we always need to land changes to master first, then change the tests afterwards to use new master features.
Not really something we can fix here (but maybe we can point it to the integration branch of bors once we have it, at least for CI)

Reply:
> Nah, I mean more the fact that we can't just point to "this directory two layers above".
> With this we always need to land changes to master first, then change the tests afterwards to use new master features.
> Not really something we can fix here (but maybe we can point it to the integration branch of bors once we have it, at least for CI)

Unfortunately we can't use relative URL's in a File URI, which is what is expected when we use the `git` command in the Cartfile.  I tried this several different ways but just couldn't get it to find it unless I put an absolute path in the URL (i.e "file:///Users/tlong/Documents/glean" works, but "file://../../.." doesn't).

Reply:
I'll file a bug to change this to point at the integration branch or something once bors is enabled.

Reply:
[Bug filed](https://bugzilla.mozilla.org/show_bug.cgi?id=1599531)

Reply:
We could, for now, change CI to run these tests only on master, thus at least making them not fail on the PR itself.

Reply:
With #531 landed you should be able to keep this as is and CI will do the right thing.